docs/plugins.md

```diff
@@ -45,6 +45,24 @@ bitops:
     filename: bitops-run      # log filename
     err: bitops.logs          # error logs filename
     path: /var/logs/bitops    # path to log folder
+    # Define the secrets to mask
+    masks:
+      - # regex to search
+        # looks for `BITOPS_KUBECONFIG_BASE64={string}`
+        search:  (.*BITOPS_KUBECONFIG_BASE64.*\=)(.*\n)
+        # replace the value part
+        replace: '\1*******\n'
+      - # looks for `The namespace kube-system exists`
+        search:  (.*The namespace )(kube-system)( exists.*)
+        #replace kube-system
+        replace: '\1*******\3'
+      - # see: https://regex101.com/r/44Ldz7/1
+        # looks for `AWS_ACCESS_KEY_ID={string}`
+        search: (AWS_ACCESS_KEY_ID=)(\S+)
+        replace: \1*******
```

Comment:
Tested the regex rules for AWS locally.

But it's also interesting that you can have unit tests and examples for the regex rule at https://regex101.com/r/44Ldz7/1

Reply:
I would move that URL to the top of the regex comments, or to the bottom, so that every regex keeps the same style of #comment / replace

Reply:
Sorry, the PR was auto-merged.

![image](https://user-images.githubusercontent.com/1533818/208943735-046b42fc-3ec5-4314-a936-7541884e75f7.png)

Though the URL was specific to the `AWS_ACCESS_KEY_ID` rule block as an example